integration-tests/src/main/java/com/datastax/oss/quarkus/CassandraNameConverterEndpoint.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus;
+
+import com.datastax.oss.quarkus.dao.nameconverters.NameConverterEntity;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/cassandra-name-converter")
+public class CassandraNameConverterEndpoint {
```

Comment:
Would it be possible to simplify the tests to use just one endpoint, one service and one DAO + name converter? I'm having a hard time trying to follow what the tests do :-)

Reply:
I was also having a problem understanding those quarkus tests mainly because of the indirection layer (rest API) but this is recommended. 
Regarding two endpoints, I would like to have that split because of the `/cassandra` is testing a code path that does not involve reflection, whereas `cassandra-name-converter` tests naming converter that uses reflection, see:
```
@RegisterForReflection
public class TestNameConverter implements NameConverter {
```
We may extend those tests in the future, so I think that split is good here. wdyt?

Reply:
But in this case, isn't testing the code path with reflection enough?

Reply:
When I was developing it there were some cases when the `testSaveAndRetrieveProduct` was passing and `shouldSaveAndRetrieveUsingCustomNameConverterThatUsesReflection` was not so I think that is a good reason to have them separately. I think that tests should verify the narrow scope of code and give more precise feedback. If we will keep only the 2nd test, we would not be verifying the 1st code path.